### PR TITLE
DOCS: Correct & expand routing docs

### PR DESCRIPTION
**Pull request by oddnoc**
- Correct syntax error in TeamController sample code
- Explain mandatory use case of $url_handlers
- Provide sample code for mandatory use case of $url_handlers

### CODE REVIEW

File: docs/en/02_Developer_Guides/02_Controllers/02_Routing.md [hunk 1]
```diff
@@ -154,12 +162,35 @@ This is useful when you want to provide custom actions for the mapping of `teams
...
+class specifies the URL pattern in `$url_handlers`. Notice that it defines 5
+parameters.
+
+
+	:::php
+	class FeedController extends ContentController {
+
+		private static $allowed_actions = array('Go');
+		private static $url_handlers = array(
+			'Go/$UserName/$AuthToken/$Timestamp/$OutputType/$DeleteMode' => 'Go'
```
Comment (dhensby) [inline docs/en/02_Developer_Guides/02_Controllers/02_Routing.md]: I like the idea of a "real world" example, but this seems quite obscure and lacking context.

Could you either provide some context (perhaps as simple as adding an action) or use a concocted example which is a bit more obvious?

**Comment by dhensby** [comment]
Thanks. Can you squash these please?

```bash
$ git reset HEAD^
$ git commit --amend --no-edit
$ git push -f
```

**Comment by oddnoc** [comment]
All squashed! (BTW, I do discrete commits to make discussion easier, with the intention of squashing at the end. Let me know if you think that's unnecessary.)

**Comment by dhensby** [comment]
> I do discrete commits to make discussion easier, with the intention of squashing at the end

Completely agree with that approach, I just add the commands to help as I don't know who does and doesn't know how to do this out of the box :)

**Comment by dhensby** [comment]
Thanks!

**Pull request merged**